README.md

```diff
@@ -24,40 +24,51 @@ cd sample-go-rabbitmq
 
 #### Install RabbitMQ via Helm
 
-Since the Helm stable repositoty was migrated to the Bitnami repository (https://github.com/helm/charts/tree/master/stable/rabbitmq), add the Bitnami repo and use it during the installation (bitnami/<chart> instead of stable/<chart>) 
+Since the Helm stable repositoty was migrated to the [Bitnami Repository](https://github.com/helm/charts/tree/master/stable/rabbitmq), add the Bitnami repo and use it during the installation (bitnami/\<chart\> instead of stable/\<chart\>)
 
 ```cli
-$ helm repo add bitnami https://charts.bitnami.com/bitnami
+helm repo add bitnami https://charts.bitnami.com/bitnami
 ```
 
 ##### Helm 3
 
 RabbitMQ Helm Chart version 7.0.0 or later
+
 ```cli
 helm install rabbitmq --set auth.username=user --set auth.password=PASSWORD bitnami/rabbitmq --wait
 ```
 
-NOTE: With RabbitMQ Helm Chart version 6.x.x or earlier, username and password should be specified with rabbitmq.username and rabbitmq.password parameters https://hub.helm.sh/charts/bitnami/rabbitmq
+**NOTES:**
 
-NOTE: if you are running the rabbitMQ image on KinD, you will run into permission issues unless you set ``volumePermissions.enabled=true``
+* If using macOS with Apple Processor (ARM architecture), the default RabbitMQ image built by bitnami does not support ARM architecture so use the [official rabbitmq image](https://hub.docker.com/_/rabbitmq) instead. To install RabbitMQ using the official rabbitmq image run the following command:
```

Comment:
I wouldn't say macOS because this applies to all ARM64 cpus. MacOS creates a vm with linux-amd64 for running docker containers